hack/collector-harvester

```diff
@@ -79,7 +77,6 @@ collect_etc ${BUNDLE_DIR}/configs/etc
 mkdir -p logs
 cd logs
 
-chroot $HOST_PATH /usr/bin/journalctl -k > kernel.log
```

Comment:
This makes the kernel log absent in the harvester. We don't have /var/log/syslog and timestamp in dmesg is hard to read.
I'd suggest not touching the harvester's collector and running the `default` collector only on an unknown os, perhaps calling `collector-default` here? https://github.com/rancher/support-bundle-kit/blob/3b32c66edb28a998d4a9323203d0aa28e7769fd1/hack/support-bundle-collector.sh#L28

Reply:
Thanks. I have update the timing to execution of the `collector-default` script to execute when no specific OS collector script is found. 

And also restore the original adjustments to the collector scripts of Harvester and k3os.